content/the-ember-times-issue-182.md

```diff
@@ -2,27 +2,32 @@
 title: The Ember Times - Issue No. 182
 authors:
   - the-crowd # replace with real authors from the author folder (add yourself if you're not there)
+  - chris-ng
 date: 2021-06-18T00:00:00.000Z
 tags:
   - newsletter
   - '2021'
 ---
 
 👋 Emberistas! 🐹
-
+Read the blog post on using the in-element helper ⛳,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [1. Section title in sentence case 🐹](section-url)
+## [Blog Post: Using the in-element helper ⛳](https://www.linkedin.com/pulse/emberjs-using-in-element-helper-faith-or/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Faith Or (@faith-or)](https://github.com/faith-or) wrote a blog post on how they migrated from `ember-wormhole` to the built-in `in-element` helper introduced in Ember 3.20.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+[Ember Wormhole](https://github.com/yapplabs/ember-wormhole) is an addon that renders a child view somewhere else in the DOM.
```

Comment:
I haven't used `ember-wormhole`, but think it's used to render something outside of the usual, parent DOM tree? If so, I think we could be more specific, in case a reader isn't familiar with the addon.

```suggestion
[Ember Wormhole](https://github.com/yapplabs/ember-wormhole) is an addon that renders a child view somewhere _outside_ the parent element in DOM.
```

Reply:
Ah I left my train of throught lemme redo this